src/GraphQL/Utilities/SchemaPrinterOptions.cs

```diff
@@ -19,5 +19,10 @@ public class SchemaPrinterOptions
         /// Indicates whether to use ',' instead of '&amp;' when inheriting a type from multiple interfaces.
         /// </summary>
         public bool OldImplementsSyntax { get; set; }
+
+        /// <summary>
+        /// Indicates whether to print descriptions as comments for compatibility with the 2016 GraphQL specification.
+        /// </summary>
+        public bool PrintDescriptionsAsComments { get; set; } = true;
```

Comment:
@Shane32 Hi! I continue to view summer emails. Is this a candidate for removing in v5?

Reply:
Fine with me, but we should mark as obsolete in v4.